mod/admin.php

```diff
@@ -1400,17 +1400,17 @@ function admin_page_site(App $a)
 		'$ostatus_not_able'	=> L10n::t("OStatus support can only be enabled if threading is enabled."),
 		'$diaspora_able'	=> $diaspora_able,
 		'$diaspora_not_able'	=> L10n::t("Diaspora support can't be enabled because Friendica was installed into a sub directory."),
-		'$diaspora_enabled'	=> ['diaspora_enabled', L10n::t("Enable Diaspora support"), Config::get('system','diaspora_enabled'), L10n::t("Provide built-in Diaspora network compatibility.")],
+		'$diaspora_enabled'	=> ['diaspora_enabled', L10n::t("Enable Diaspora support"), Config::get('system', 'diaspora_enabled', $diaspora_able), L10n::t("Provide built-in Diaspora network compatibility.")],
```

Comment:
Is the variable name correct? `$diaspora_able`

Reply:
Jepp, see line 1401. It is set whenever Diaspora is able to be enabled on the server.